.github/workflows/checkJSON.yml

```diff
@@ -41,4 +42,21 @@ jobs:
            sudo apt-get install jsonlint
            jsonlint-php ${JSON_FILES}
            python3 scripts/checkJSONConsistency.py ${JSON_FILES}
+           result=$(cat coverage.txt)
```

Comment:
Could we fill coverage.txt with a message when everything is fine ?
So the action post that everything is alright ?

Reply:
ok your solution is better.